src/viam/ur/module/ur_arm.hpp

```diff
@@ -47,6 +47,7 @@ void sampling_func(std::vector<trajectory_sample_point>& samples, double duratio
     samples.push_back(f(duration_sec, step));
 }
 
+std::vector<std::string> validate_config(const ResourceConfig& cfg);
```

Comment:
this should probably be a private method to `URArm`. Methods defined in this unnamed namespace likely to migrate to a `utils` module.